tasks/slurmctld.yml

```diff
@@ -4,6 +4,7 @@
   ansible.builtin.package:
     name: "{{ __slurm_packages.slurmctld }}"
     state: "{{ 'latest' if slurm_upgrade else 'present' }}"
+  when: slurm_apt_repository is not defined
```

Comment:
it is in the same logic not to have to install slurm packages twice when it is compiled.

Reply:
So if one compile their own packages with separate `slurm` and `slurmctld`, then define their own APT repository in `slurm_apt_repository`, the package `slurmctld` will never be installed? That's unexpected.

Anyway if a package is already installed, the task will not install it twice, it will just move to the next task.

Reply:
unfortunately yes in the case where the packages are separate it will not pass I will leave the initial vversion in the installtion of the packages